ccnmtldjango/template/+package+/settings_production.py_tmpl

```diff
@@ -3,13 +3,13 @@ from settings_shared import *
 import os
 
 TEMPLATE_DIRS = (
-    os.path.join(os.path.dirname(__file__), "templates"),
```

Comment:
General comment: Do these settings (TEMPLATE_DIRS, MEDIA_ROOT, STATICMEDIA_MOUNTS) need overridden in production & staging settings when they are now generically constructed in settings_shared?